etc/config/javascript.amazon.properties

```diff
@@ -0,0 +1,16 @@
+compilers=javascript
+defaultCompiler=javascript
+
+supportsBinary=false
+interpreted=true
+compilerType=v8
+objdumper=
+demangler=
+postProcess=
+options=
+supportsExecute=false
+stubText=function add(a, b) { return a + b; } for (var i = 0; i < 10; i++) { add(i, 1) }
+
+compiler.javascript.exe=/opt/compiler-explorer/d8-11.1/out/x64.release/d8
```

Comment:
Even though you say you'll never ever want to support more than one version of javascript...I'm less sure. If we have a floating version of this we should call it "trunk" or similar, else we should name it here both in the ID and in the `name`.  Honestly, URLs are forever and we don't want to use up "javascript" to mean "one particular random version of d8 we installed once"

Reply:
I'm updating the PR to remove the reference of `javascript` from the compiler configurations and instead use `v8`. This will also leave room for people to add other javascript engines in the future. Additionally, I'm adding a group with `trunk` and the latest version. Just so there are at least two versions available on the initial merge and if it is found at a later time that more versions are desirable they can be added easily.